fdbclient/include/fdbclient/StorageCheckpoint.h

```diff
@@ -52,6 +52,7 @@ struct CheckpointMetaData {
 	std::vector<UID> src; // Storage server(s) on which this checkpoint is created.
 	UID checkpointID; // A unique id for this checkpoint.
 	int16_t state; // CheckpointState.
+	std::string bytesSampeFile;
```

Comment:
nit: maybe `Optional<std::string> bytesSampeFile`, it is present only when there is a metadata file?